DESCRIPTION

```diff
@@ -1,14 +1,26 @@
 Package: vscDebugger
 Title: Support for Visual Studio Code Debugger
 Version: 0.4.5
-Authors@R: 
-    person(given = "Manuel",
-           family = "Hentschel",
-           role = c("aut", "cre"),
-           email = "Manuel.Hentschel@outlook.com")
+Authors@R: c(
+    person(
+        given = "Manuel",
+        family = "Hentschel",
+        role = c("aut", "cre"),
+        email = "Manuel.Hentschel@outlook.com"
+    ),
+    person(
+        given = "Kun",
+        family = "Ren",
+        role = "ctb"
```

Comment:
Would be nice if my email is added like https://github.com/REditorSupport/languageserver/blob/master/DESCRIPTION#L14.